updater/osv/osv.go

```diff
@@ -554,24 +558,40 @@ func (e *ecs) Insert(ctx context.Context, skipped *stats, name string, a *adviso
 			}
 			// This does some heavy assumptions about valid inputs.
 			ranges := make(url.Values)
```

Comment:
does this need the same kind of treatment?

Reply:
https://api.osv.dev/v1/vulns/PYSEC-2008-3 Good catch, it appears like yes

Reply:
Updated